soundfile.py

```diff
@@ -305,12 +377,22 @@ def read(file, frames=-1, start=0, stop=None, dtype='float64', always_2d=False,
     with SoundFile(file, 'r', samplerate, channels,
                    subtype, endian, format, closefd) as f:
         frames = f._prepare_read(start, stop, frames)
-        data = f.read(frames, dtype, always_2d, fill_value, out)
+        data = f.read(
```

Comment:
why was this reformatted?

Reply:
Because of overload. Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/8b5de1310489eb723d177945a651e0fa9ea18419